DTDL/publication.json

```diff
@@ -0,0 +1,100 @@
+{
```

Comment:
Publications could previously reference a contract. Not saying it's important, but maybe we still need it?

Reply:
if we add it we should inline the TOU into the publication? or would you like a separate entity

Reply:
Till now it was defined per publication (aka not re-used), inline would be fine for me.